spatialos-sdk-code-generator/src/bin.rs

```diff
@@ -0,0 +1,23 @@
+extern crate spatialos_sdk_code_generator;
+
+use spatialos_sdk_code_generator::generator;
+use spatialos_sdk_code_generator::schema_bundle;
+use std::env::args;
+use std::fs::File;
+use std::io::Read;
+use std::io::Write;
+
+pub fn main() {
+    let input_filename = args().nth(1).unwrap();
+    let output_filename = args().nth(2).unwrap();
```

Comment:
Rather than parsing arguments manually, I recommend using a crate like [structopt](https://crates.io/crates/structopt) to handle the parsing logic for you. It'll make the code cleaner, and it'll provide a better UX by automatically generating help text and the like.